docs/introduction.md

```diff
@@ -59,3 +59,9 @@ After the promise is settled, a second action will be dispatched. If the the pro
   }
 }
 ```
+
+The default promise status suffix can be imported from this module.
+
+```js
+import { PENDING, FULFILLED, REJECTED } from 'redux-promise-middleware'
```

Comment:
You can rename the constants that are destructed as they're just the values in the array. Then you can keep the exported constants less verbose.

```js
export const PENDING = 'PENDING';
export const FULFILLED = 'FULFILLED';
export const REJECTED = 'REJECTED';

const defaultTypes = [PENDING, FULFILLED, REJECTED];

// ...

const [
  _PENDING,
  _FULFILLED,
  _REJECTED
] = promiseTypeSuffixes

// make sure you update the places these are used to have the _ prefix.
```